docs/learn/learn-account-generation.md

```diff
@@ -214,6 +239,16 @@ off the computer you're using. You should not store it in cloud storage, email i
 You can use this backup file to restore your account. This backup file is not readable unless it is
 decrypted with the password.
 
+### Reset password
+
+To reset the password of an account created with Polkadot-JS you just need to go in the "Accounts" tab, click the icon with three vertical dots on your account and select "Change this account's password".
+
+:::info
+
+If you create an account first using Polkadot-JS, and then you add it to the browser extension, to change the passowrd of such account you need to follow the [guidelines for the browser extension](#reset-password).
```

Comment:
This section is named as reset-password as well. The link works and points to the reset-password subsection from the JS Extension section. In general, the recommendation is to give unique titles to sections and subsections

Reply:
I specified different subtitles. See latest push.